libraries/debian_helpers.rb

```diff
@@ -6,27 +6,26 @@ module Helpers
     module Debian
       def enable_auto_updates_debian
         package 'debconf-utils' do
-          action :install
+          action :upgrade
         end
         package 'unattended-upgrades' do
-          response_file lazy { 'unattended-upgrades.seed.erb' }
-          action :install
+          response_file 'unattended-upgrades.seed.erb'
+          action :upgrade
         end
-        unattended_upgrade_file = '/etc/apt/apt.conf.d/50unattended-upgrades'
+        unattended_upgrades_file = '/etc/apt/apt.conf.d/50unattended-upgrades'
         execute 'rackspace_allowed_origins' do
-          command "perl -pi -e 's/(Allowed-Origins\s*{)/$1\n\t\"serveragent main\";\n\t\"cloudmonitoring main\";/;' #{unattended_upgrade_file}"
+          command "perl -pi -e 's/(Allowed-Origins\s*{)/$1\n\t\"serveragent main\";\n\t\"cloudmonitoring main\";/;' #{unattended_upgrades_file}"
           only_if { rackspace? }
           action :run
         end
         # New format(Currently only Deb 7)
         execute 'rackspace_unattended_upgrade' do
-          command "perl -pi -e 's/(Unattended-Upgrade::Origins-Pattern\s*{)/$1\n\t\"o=serveragent,a=main\";\n\t\"o=cloudmonitoring,a=main\";/;' #{unattended_upgrade_file}"
+          command "perl -pi -e 's/(Unattended-Upgrade::Origins-Pattern\s*{)/$1\n\t\"o=serveragent,a=main\";\n\t\"o=cloudmonitoring,a=main\";/;' #{unattended_upgrades_file}"
           only_if { rackspace? }
           action :run
         end
         execute 'enable_auto_updates' do
           command 'echo unattended-upgrades unattended-upgrades/enable_auto_updates boolean true | debconf-set-selections'
-          only_if 'debconf-get-selections | grep unattended-upgrades | grep boolean | grep false'
```

Comment:
It would be nice if the resource didn't report a change on every run, and cause a notification on every run. I assume that was why the guard was there -- I get a lot of folks complaining about resources reporting an updated state when they haven't actually made a change from the initial state of a system. Do you think that's a problem here?